openupgrade_scripts/scripts/website_sale/14.0.1.0/post-migration.py

```diff
@@ -1,10 +1,40 @@
 # Copyright (C) 2021 Open Source Integrators <https://www.opensourceintegrators.com/>
 # Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import re
 
 from openupgradelib import openupgrade
 
 
+def extract_custom_product_page_term_conditions(env):
+    """Replace Terms and Conditions content in the new v14 template so as not to lose
+    content from previous versions if it has been customised."""
+    product_custom_text_view = env["ir.ui.view"].search(
+        [("key", "=", "website_sale.product_custom_text"), ("website_id", "=", False)]
```

Comment:
If you want the main view, grab it by xml_id ref

Reply:
thanks!